.jenkins/continuous.groovy

```diff
@@ -95,8 +95,15 @@ pipeline {
                                 -DKokkos_ENABLE_TESTS=ON \
                                 -DKokkos_ENABLE_BENCHMARKS=ON \
                                 -DKokkos_ENABLE_HIP=ON \
+                                -DKokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE=ON \
+                                -DCMAKE_INSTALL_PREFIX=${PWD}/../install \
                               .. && \
-                              make -j16 && ctest --no-compress-output -T Test --verbose'''
+                              make -j16 && ctest --no-compress-output -T Test --verbose && \
+                              make install && \
+                              export CMAKE_PREFIX_PATH=${PWD}/../install && \
+                              cd ../example/build_installed && \
+                              rm -rf build && mkdir -p build && cd build && \
+                              cmake -DCMAKE_CXX_STANDARD=20 -DExamples_CMAKE_LANGUAGE=HIP .. && make -j8'''
```

Comment:
I disabled running the examples since the container complains about a shared fortran library that is a part of flang can not be found. I think installing flang from scratch in the container is a bit of an overkill